### PR TITLE
User/opa/release notes 105

### PR DESCRIPTION
**Pull request by DoxaPap**

### CODE REVIEW

File: portal/portal-sprint-105.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+The Quotes Page in the Shop in Portal has been reworked and now the user experience has been improved. The performance of the page has increased significantly, and a bigger amount of data can now be handled more efficiently. The general functionalities are identical to the past, but a search bar and pagination functionality have now been added.
+
+![image](images/sprint-105/invoicepage.png)
+
+### Reworked CashBox Edit Page
+
+The CashBox Edit Page in Portal has been reworked and now the user experience has been improved. The performance of the page has increased significantly, and a bigger amount of data can now be handled more efficiently. The general functionalities are identical to the past, and now the refactored dropdown Box is able to deal with more general scenarios. Aside from this, updating the selected outlet on the Cashbox Edit page, now also updates the outlet filter. 
+
+![image](images/sprint-105/cashbox-edit-page.png)
```
Comment (FlorianStadlberger) [inline portal/portal-sprint-105.md:49]: The image is stretched. We should either fix the aspect ratio or just remove it, as it does not show much.

File: portal/portal-sprint-103.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
```
Comment (FlorianStadlberger) [inline portal/portal-sprint-103.md:1]: Why is this file in this PR?
If there are any changes, they should be applied to this file in the 103 release note branch & PR.

File: portal/portal-sprint-105.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+Status
+Docs
+Contact GitHub
+Pricing
+API
+Training
+Blog
+About
+
+
```
Comment (FlorianStadlberger) [inline portal/portal-sprint-105.md:73]: Could do without some empty spaces 😄

**Review by FlorianStadlberger — CHANGES_REQUESTED** [review]

File: portal/portal-sprint-103.md (hunk 2 above)
Comment (DoxaPap) [reply portal/portal-sprint-103.md]: I honestly don't know how i managed to do that :D

File: portal/portal-sprint-105.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+### Reworked CashBox Edit Page
+
+The CashBox Edit Page in Portal has been reworked and now the user experience has been improved. The performance of the page has increased significantly, and a bigger amount of data can now be handled more efficiently. The general functionalities are identical to the past, and now the refactored dropdown Box is able to deal with more general scenarios. Aside from this, updating the selected outlet on the Cashbox Edit page, now also updates the outlet filter. 
+
+![image](images/sprint-105/cashbox-edit-page.png)
+
+### Display Subscription Period
+
+PosDealers -aside from being able to extend and cancel subscriptions- are now able to know exactly when those subscriptions started or ended, when navigating through the Subscriptions Table Page. This is an important improvement, as in the past PosDealers had no idea about the runtime of subscriptions. Now, while loading subscription data the Start and Expire Date of products are calculated and are being displayed in separate columns in the Subscriptions Page Table.
```
[the remark this answers is not in the document]
Comment (DoxaPap) [reply portal/portal-sprint-105.md:53]: https://www.thepunctuationguide.com/em-dash.html#:~:text=Dashes%20are%20considered%20less%20formal,content%20more%20subtly%2C%20use%20parentheses.&text=When%20used%20in%20place%20of,a%20single%20dash%20is%20used.

not changing it lol

**Review by FlorianStadlberger — APPROVED** [review]

**Pull request merged**